README.md

```diff
@@ -63,6 +63,8 @@ $writtenEvents = $client->writeEvents([
     ],
   ),
 ]);
+
+iterator_to_array($writtenEvents);
```

Comment:
Zwei Gedanken hierzu:

1. Da die Funktion ja immer dasselbe macht, fände ich es sinnvoller, die Funktion einmal zentral als Hilfsfunktion vorzustellen, statt sie immer wieder in die Beispiele einzubauen. Was hältst Du davon?
2. Ich find's etwas ungewöhnlich, dass die Funktion augenscheinlich nichts zurückgibt – fehlt hier das Entgegennehmen des Rückgabewerts, oder wie funktioniert das? Bzw. anders gefragt: Könnten wir die Funktion nicht vielleicht so umbauen, dass sie ein Array zurückliefert?